### PR TITLE
issue-2 Install: express, react, typescript, webpack and add .npmrc …

### PR DESCRIPTION
**Pull request by Elfant**

### CODE REVIEW

**Comment by Elfant** [comment]
[Initialize node project.](https://trello.com/c/EYxY9Xqe/2-initialize-node-project)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```
Comment (krzysztofbukowski) [inline .gitignore]: Missing new line at the end of file

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v12.13
```
Comment (krzysztofbukowski) [inline .npmrc]: Missing new line at the end of file

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "library",
+  "version": "1.0.0",
```
Comment (krzysztofbukowski) [inline package.json]: Let's change the version to 0.1.0. We'll set it as 1.0.0 once the MVP is ready :D

File: package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/bukowski-se/library/issues"
+  },
+  "homepage": "https://github.com/bukowski-se/library#readme",
+  "dependencies": {
+    "@types/node": "^12.12.7",
+    "@types/react": "^16.9.11",
+    "express": "^4.17.1",
```
Comment (krzysztofbukowski) [inline package.json]: Let's also add `@types/express`

**Pull request closed without merging**